client/.npmrc

```diff
@@ -1,2 +1,7 @@
 # TODO: turn this off at some point
 legacy-peer-deps = true
+
+# PNPM
+link-workspace-packages = deep
+ignore-workspace-root-check = true
+shamefully-hoist = true # wish this wouldn't be required, but build tools won't cooperate
```

Comment:
It's hard to explain, but I'll try:

NPM (and/or Lerna) will hoist a workspaces dependencies all the way to the root. If a package depends on `xyz@1.0.0`, the root will get a package installed in `node_modules` named `xyz`. Hoisting!

But, PNPM doesn't directly do that, because that opens up the issue of _spoooky phantom dependencieeessss_ which basically means that if another package which depends on `xyz` can just nab it from `node_modules` at the monorepo root, _without actually depending on it inside of the package's `package.json`_.

PNPM instead shoves all of the hoisted dependencies into `.pnpm` or something in `node_modules` and then symlinks the hard, actually listed dependencies for the root monorepo package into `node_modules`. This preserves "what you expect", but sometimes build tools don't resolve a subpackage's dependencies, for some god damn reason, and instead try to resolve them from the monorepo root. By "shamefully hoisting", PNPM won't make a strict `node_modules` and the naughty build tools will find the packages their looking for in the root `node_modules`.

Clear as mud?